### PR TITLE
Add alpine packages to INSTALL.md

### PR DESCRIPTION
**Pull request by svrnm**
As discussed in #1950 a minor update to INSTALL.md to also mention the alpine packages.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -320,6 +320,10 @@ with `vcpkg install opentelemetry-cpp` and follow the then displayed
 descriptions. Please see the vcpkg project for any issues regarding the
 packaging.
 
+If you are using [alpine linx](https://www.alpinelinux.org/) you can install
```
Comment (marcalff) [inline INSTALL.md:323]: There is a trailing space, making the format CI fail.

Also, `alpine linux` for the link name ?

Comment (marcalff) [inline INSTALL.md:323]:
```suggestion
If you are using [alpine linux](https://www.alpinelinux.org/) you can install
```

**Review by marcalff — APPROVED** [review]
LGTM, thanks the the fix, and for the packages.

**Review by esigo — APPROVED** [review]
LGTM
Thanks for the package :)

**Pull request merged**